npm/private/npm_translate_lock_generate.bzl

```diff
@@ -217,7 +216,7 @@ def npm_link_targets(name = "node_modules", package = None):
     npm_link_all_packages_bzl = [
         """\
 # buildifier: disable=function-docstring
-def npm_link_all_packages(name = "node_modules", imported_links = []):
```

Comment:
I guess one question would be... should we keep the `name` param for consistency across rules but throw if it is not the default? Does that help IDEs or anything like that? 🤷